### ISSUE TITLE
Option for last commented date when searching

### ISSUE THREAD

**Issue by boothym** [ENT8R/NotesReview#37]
Using the "to" date in the search function is useful for finding old notes (i.e. set to 1/1/2014 or similar) - however some of these notes have been commented on so they're not really old anymore. 

Would it be possible to have an option to use the last commented date in when searching?

**Comment by ENT8R** [comment]
I also think it is a little bit confusing to sort the results by their `updated_at` value instead of their creation date...
So I opened a new issue regarding this functionality: https://github.com/openstreetmap/openstreetmap-website/issues/2380

**Comment by boothym** [comment]
Not sure you've understood the issue - sorry if I've not made myself clear enough. 😃 

Take this example: 

I want to find all notes which have **not been updated after 1/1/2014**, so I set the to date and then search. Useful to find notes which nobody has bothered about for years.

https://ent8r.github.io/NotesReview/?limit=100&to=2014-01-01&map=13%2F57.126%2F-2.0958

However, the map shows me notes which were **created before 1/1/2014**, even though each of them has been updated in the past 6 years. You could have a note like this: https://www.openstreetmap.org/note/852656 where it is being actively updated, so it's not the created date that I'm worried about.

Basically, I would like to have a choice between searching for the date the note was created, and when it was last updated.

**Comment by ENT8R** [comment]
So, do you want to have an option to only show notes which have not received a single comment in the time between 1/1/2014 and now? Meaning that the last time the note has been updated is before the 1/1/2014.
In that case it would be fixed once https://github.com/openstreetmap/openstreetmap-website/pull/2381 is merged into the main OpenStreetMap website and the changes are implemented in the interface of NotesReview. Then it will be possible to select a date (e.g. 1/1/2014) and check a box to only show notes which have been updated **before** that date. Right now it is only possible to select a date and show the notes which have been created **before** that date...

**Comment by ENT8R** [comment]
The pull request just got merged yesterday, so with https://github.com/ENT8R/NotesReview/commit/e5404b03b949bc3430ba8084a8adfb7814a55e09 I implemented the option to sort notes similar to how Github does it 🎉:
- Newest
- Oldest
- Recently updated
- Least recently updated
The date filtering is still possible like it was before. So I hope your use case is covered by this enhancement 😄
